modules/F50_base_aggregator.sh

```diff
@@ -132,10 +132,14 @@ output_details() {
     echo "python_vulns;\"$S21_PY_VULNS\"" >> "$CSV_LOG_FILE"
     DATA=1
   fi
-  if [[ "$S22_PHP_VULNS" -gt 0 ]]; then
+  if [[ "$S22_PHP_VULNS" -gt 0 ]] || [[ "$S22_PHP_INI_ISSUES" -gt 0 ]]; then
```

Comment:
Probably we should separate these two into two if statements
if PHP_VULNS
bla
fi
if PHP_INI_ISSUES
bla
fi

otherwise we can get reports with 0 values